Exercises/strings/strings.js

```diff
@@ -16,4 +16,41 @@ console.log(`My favourite animal is cat ${favoriteCat}`);
 const favoriteWriter = 'herbert wells';
 const wordsDivisorIndex = favoriteWriter.indexOf(' ');
 const properFavoriteWriter = `${favoriteWriter[0].toUpperCase()}${favoriteWriter.slice(1, favoriteWriter.indexOf(' '))} ${favoriteWriter[wordsDivisorIndex + 1].toUpperCase()}${favoriteWriter.substring((wordsDivisorIndex + 2), favoriteWriter.length)}`;
-console.log(properFavoriteWriter);
\ No newline at end of file
+console.log(properFavoriteWriter);
+
+
+// # Additional
+
+// 1. Declare a function which will receive password string. Check is password strong enough:
+//   - should contain at least one capital letter;
+//   - length no less than 8 characters.
+//   - should contain at leas one special character.
+
+function validatePassword(password) {
+  if (password.length < 8) {
+    return false;
+  } else if (!password.match(/[A-Z]/)) {
+    return false;
+  } else if (!password.match(/[!@#$%^&*(),.?":{}|<>]/)) {
+    return false;
+  }
+  return true;
```

Comment:
Suppose you have 10 rules instead of just 3. Try to write this function with 3 rules using `Array.prototype.some` or `Array.prototype.every`